packages/api-bindings/src/lens/__helpers__/queries/profile.ts

```diff
@@ -0,0 +1,121 @@
+import { MockedResponse } from '@apollo/client/testing';
+
+import {
+  FollowersDocument,
+  FollowersVariables,
+  FollowingDocument,
+  FollowingVariables,
+  MutualFollowersDocument,
+  MutualFollowersVariables,
+  PaginatedResultInfo,
+  Profile,
+  ProfilesData,
+  ProfilesDocument,
+  ProfilesVariables,
+} from '../../graphql/generated';
+import { mockPaginatedResultInfo } from '../fragments';
+
+/**
+ * All paginated profile responses have the same shape
+ */
+function mockAnyPaginatedProfilesResponse<V, Q>({
```

Comment:
I decided to organize mocks a bit better, to avoid having one single huge file with everything.
I also extracted a reusable response shape into a helper function, used only internally. wdyt?

Reply:
Much better, great shout! Thanks Kris